### DISCUSSION TITLE
Minimum required Node version?

### DISCUSSION THREAD

**Discussion by slowfound** [jeffreylanters/react-unity-webgl#225] [Question and Answer]
Hey everybody. For dependency reasons, I have to run my app in Node 8.11.2 and I'm having trouble adding this package to my app. Is there minimum required Node version?

**Comment by jeffreylanters** [comment]
To be honest, not really as far as I know about. TypeScript is configured to compile to EsmaScript version 5. If your node version supports that, all should be good. Another solution would be to wrap the module without your own project, and compile it to an even lower EsmaScript version. I'm not supporting it natively and don't test on these lower versions, but feel free to fiddle around. I hope this informs you enough. Happy coding!
